glam/api/views.py

```diff
@@ -46,7 +61,14 @@ def updates(request):
     )
 
 
-def get_firefox_aggregations(request, **kwargs):
+def get_firefox_aggregations(source, request, **kwargs):
+    if source == "BigQuery":
+        return get_firefox_aggregations_from_bq(request, **kwargs)
+    else:
+        return get_firefox_aggregations_from_pg(request, **kwargs)
+
+
+def get_firefox_aggregations_from_pg(request, **kwargs):
```

Comment:
Database routers aren't supported. And the `_from_pg` function is there only to ensure easy rollback. Therefore this helper function will go away together with the `_from_pg` variant once we are confident we're sticking to BQ. 

The goal is to be able to do a quick switch and investigate any problems later locally in case of some disruption in prod caused by this change.